src/makeCalculator.js

```diff
@@ -37,7 +37,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+
+    add(a) {
+      calc.result += a;
```

Comment:
if I use `this` instead of `calc`, operate method can't access  `result` 
![image](https://github.com/mate-academy/js_advanced_calculator/assets/147757809/c0a6b125-d60c-43c0-abb0-1c1c1c08900e)